src/modules/load_scores.js

```diff
@@ -0,0 +1,7 @@
+export default function loadScores(htmlContainer, recentScores) {
+  recentScores.forEach((score) => {
+    const listItem = document.createElement('li');
+    listItem.textContent = `${score.name}: ${score.score}`;
+    htmlContainer.appendChild(listItem);
+  });
+}
```

Comment:
- [ ] Great job with the project as everything is working as intended, but as one of the learning objectives is to learn proper ES6 syntax let's try using an arrow function instead of the function keyword
![image](https://user-images.githubusercontent.com/73140461/151049106-ae898249-662c-4821-a295-d05f847245a2.png)

So instead of
<pre>
export default function loadScores(htmlContainer, recentScores) {
  ...your function
}
</pre>

let's try:
<pre>
const loadScores = (htmlContainer, recentScores) => {
 ...your function
}

export default loadScores

</pre>